Wabbajack.Lib/Downloaders/AbstractDownloadState.cs

```diff
@@ -7,6 +7,17 @@
 
 namespace Wabbajack.Lib.Downloaders
 {
+    public interface IMetaState
```

Comment:
Might consider just making this readonly?  Eg.. removing the `set` halves.
I imagine most users of this interface just want to get things.  Might make things simpler, like with the NexusState overriding the `Name` property.. it doesn't also have to define the set side

Reply:
Just "bumping" this comment, as it's applicable again.  I think the interface would be better as a get-only interface.  No need to require implementers to provide a `set`.  Users of `IMetaState` will likely only ever be reading info.. not setting it.

The reason this might be important is that requiring a `set` makes it difficult for the classes implementing the interface if they'd prefer the fields to be derivative (eg, driven by some other internal data that is queried/processed to give the result).

Something like this wouldn't work if set was required:
```
public string Name => someDataStructure.Find("name").Value;
```